docs/src/visualization.md

```diff
@@ -0,0 +1,195 @@
+# Visualization
+
+We provide some simple plotting features to help users diagnose convergence in
+the algorithm. Through
+
+- Visualizing Priors
+- Visualizing slices of parameter ensembles over iterations (or algorithm time) over input space
+- Visualizing the loss function or other computed error metrics from (`compute_error!`)[@ref]
```

Comment:
Sorry, I didn't realize you were going to verbatim paste into .md. 

- The `compute_error!` doesn't render for me
- Priors -> priors
-